src/main/java/com/spotinst/sdkjava/model/api/azure/statefulNode/ApiStatefulNodeResourceNetworkInterfaces.java

```diff
@@ -0,0 +1,149 @@
+package com.spotinst.sdkjava.model.api.azure.statefulNode;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.spotinst.sdkjava.client.rest.IPartialUpdateEntity;
+import com.spotinst.sdkjava.model.bl.azure.statefulNode.LaunchSpecNetworkSecurityGroupConfiguration;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("PartialUpdateEntityFilter")
+public class ApiStatefulNodeResourceNetworkInterfaces implements IPartialUpdateEntity {
+
+    @JsonIgnore
+    private Set<String>                                         isSet;
+    private String                                              name;
+    private String                                              resourceGroupName;
+    private Boolean                                             isPrimary;
+    private ApiLaunchSpecNetworkSecurityGroupConfiguration      networkSecurityGroup;
+    private List<ApiStatefulNodeResourceNetworkIpConfigurations>networkIpConfigurations;
```

Comment:
no space b/w variable name and variable type.
variable name should be changed to ipConfigurations.